btree/btree.c

```diff
@@ -5221,7 +5221,13 @@ void m0_btree_del_credit(const struct m0_btree  *tree,
 	node_free_credit(tree->t_desc->t_root, ksize, vsize, &cred);
 	btree_callback_credit(&cred);
 	m0_be_tx_credit_mul(&cred, MAX_TREE_HEIGHT);
+
+	/* Credits for deleting record from the node. */
+	node_rec_del_credit(tree->t_desc->t_root, ksize, vsize, &cred);
+	btree_callback_credit(&cred);
+
 	m0_be_tx_credit_mac(accum, &cred, nr);
+
```

Comment:
Please remove extra line before merging changes.

Reply:
I missed this comment before merging, I have requested @ydb242 to make this change in his PR as this is a small change.